test/new-e2e/tests/installer/all_packages_test.go

```diff
@@ -55,6 +62,14 @@ var (
 	}
 )
 
+var (
+	packagesConfig = []testPackageConfig{
+		{name: "datadog-installer", defaultVersion: fmt.Sprintf("pipeline-%v", os.Getenv("CI_PIPELINE_ID")), registry: "669783387624.dkr.ecr.us-east-1.amazonaws.com", auth: "ecr"},
+		{name: "datadog-agent", defaultVersion: fmt.Sprintf("pipeline-%v", os.Getenv("CI_PIPELINE_ID")), registry: "669783387624.dkr.ecr.us-east-1.amazonaws.com", auth: "ecr"},
+		{name: "apm-inject", defaultVersion: "pipeline-34163111", registry: "669783387624.dkr.ecr.us-east-1.amazonaws.com", auth: "ecr"},
```

Comment:
Doesn't really matter but for consistency:
```suggestion
		{name: "datadog-apm-inject", defaultVersion: "pipeline-34163111", registry: "669783387624.dkr.ecr.us-east-1.amazonaws.com", auth: "ecr"},
```

Reply:
doing it in next PR